api/src/ddoc-extraction.js

```diff
@@ -120,7 +120,7 @@ const findUpdatedAppcache = ddoc => {
 
 const findUpdated = ddoc => {
   return Promise.all([
-    findUpdatedDdocs(ddoc.deploy_info),
+    findUpdatedDdocs(ddoc.deploy_info || null),
```

Comment:
At first glance it's confusing as to why this would improve anything - I assume `null` passes the `isEqual` check but `undefined` doesn't? Maybe just add a comment to explain?

Reply:
The difference is of null vs undefined. The null would actually be saved in the DB. So on next update, it will compare null with null, vs non-existent field vs undefined (which fails the comparison),

Reply:
Alternatively, I can check if deploy_info is undefined and only set the value in medic-client when  either old or new value are defined. I didn't imagine that as looking pretty. 
What do you think?

Reply:
Yeah I think that'd be easier to read, and as a bonus you wouldn't save it as null the first time through.

I'm thinking update line 54 from:

```js
      // update the deploy info in the medic-client ddoc
      if (newDdoc._id === CLIENT_DDOC_ID) {
        newDdoc.deploy_info = deploy_info;
      }
```

to

```js
      // update the deploy info in the medic-client ddoc
      if (deploy_info && newDdoc._id === CLIENT_DDOC_ID) {
        newDdoc.deploy_info = deploy_info;
      }
```

Would that work?

Reply:
I changed it to:

```
if (newDdoc._id === CLIENT_DDOC_ID && (deploy_info || newDdoc.deploy_info)) {        
  newDdoc.deploy_info = deploy_info;
}
```

so it will overwrite `old` Horti deployments :)

Reply:
Awesome.